tests/test_channel.cpp

```diff
@@ -41,230 +41,230 @@
-}
-
-TEST_F(TestChannel, RecentChannel)
-{
-    auto channel = std::make_shared<RecentChannel<int>>(2);
-
-    channel::Ingress<int>& ingress = *channel;
-    channel::Egress<int>& egress   = *channel;
-
-    channel->await_write(42);
-    ingress.await_write(2);
-    ingress.await_write(-2);
-
-    int i;
-    egress.await_read(std::ref(i));
-    EXPECT_EQ(i, 2);
-    egress.try_read(std::ref(i));
-    EXPECT_EQ(i, -2);
-
-    /*
-    auto f = userspace_threads::async([&] {
-        boost::this_fiber::sleep_for(std::chrono::milliseconds(100));
-        ingress.close_channel();
-    });
-
-    auto s = std::chrono::system_clock::now();
-    egress.await_read(std::ref(i));
-    auto e = std::chrono::system_clock::now();
-    auto t = std::chrono::duration<double>(e - s).count();
-
-    EXPECT_GE(t, 0.1);
-    */
-}
-
-TEST_F(TestChannel, OnComplete) {}
-
-TEST_F(TestChannel, AwaitWriteOverloads)
-{
-    auto channel = std::make_shared<BufferedChannel<CopyMoveCounter>>(4);
-
-    CopyMoveCounter output;
-
-    auto check_counter =
-        [](const CopyMoveCounter& test_val, size_t copy_count, size_t move_count, bool was_copied, bool was_moved) {
-            EXPECT_EQ(test_val.copy_count(), copy_count);
-            EXPECT_EQ(test_val.move_count(), move_count);
-            EXPECT_EQ(test_val.was_copied(), was_copied);
-            EXPECT_EQ(test_val.was_moved(), was_moved);
-        };
-
-    // === xValue ===
-    channel->await_write(CopyMoveCounter());
-
-    channel->await_read(std::ref(output));
-    // Old value move in, new value move out
-    check_counter(output, 0, 2, false, false);
-
-    // === lValue reference ===
-    CopyMoveCounter l_value;
-    channel->await_write(l_value);
-
-    channel->await_read(std::ref(output));
-    // Old value was copied
-    check_counter(l_value, 0, 0, true, false);
-    // New value move in and move out
-    check_counter(output, 1, 2, false, false);
-
-    // === lValue move ===
-    CopyMoveCounter l_value_move;
-    channel->await_write(std::move(l_value_move));
-
-    channel->await_read(std::ref(output));
-    // Old value was copied
-    check_counter(l_value_move, 0, 0, false, true);
-    // New value move in and move out
-    check_counter(output, 0, 2, false, false);
-
-    // === const lValue ===
-    const CopyMoveCounter const_l_value;
-    channel->await_write(const_l_value);
-
-    channel->await_read(std::ref(output));
-    // Old value was copied
-    check_counter(const_l_value, 0, 0, true, false);
-    // New value move in and move out
-    check_counter(output, 1, 2, false, false);
-
-    // === const rValue ===
-    const CopyMoveCounter& const_r_value = CopyMoveCounter();
-    channel->await_write(const_r_value);
-
-    channel->await_read(std::ref(output));
-    // Old value was copied
-    check_counter(const_r_value, 0, 0, true, false);
-    // New value move in and move out
-    check_counter(output, 1, 2, false, false);
-}
+// class TestChannel : public ::testing::Test
```

Comment:
Is this intentional?

Reply:
No, was just trying to force a lot of coverage changes